ios/browser/api/opentabs/brave_opentabs_api.mm

```diff
@@ -100,12 +104,20 @@ - (id)copyWithZone:(NSZone*)zone {
     openDistantSession.sessionTag = [self.sessionTag copy];
     openDistantSession.modifiedTime = [self.modifiedTime copy];
     openDistantSession.deviceFormFactor = self.deviceFormFactor;
-    openDistantSession.tabs = [self.tabs copy];
+    openDistantSession.tabs = [self.mTabs copy];
   }
 
   return openDistantSession;
 }
 
+- (NSArray<IOSOpenDistantTab*>*)tabs {
+  return [self.mTabs copy];
+}
+
+- (void)setTabs:(NSArray<IOSOpenDistantTab*>*)tabs {
+  self.mTabs = [tabs mutableCopy];
```

Comment:
nit: Not a huge fan of this being exposed and modifiable from outside.
I'd have probably went with the filtering being done in this file, but it's fine.